community/README.md

```diff
@@ -11,3 +11,5 @@ This directory contains plugins contributed by community members.
 #### Path Traversal
 *   [Apache HTTP Server Path traversal and disclosure (CVE-2021-41773) Detector](https://github.com/google/tsunami-security-scanner-plugins/tree/master/community/detectors/apache_http_server_cve_2021_41773)
 
+#### Remote Code Execution (RCE)
+*   [SaltStack Salt-API Unauthenticated Remote Command Execution (CVE-2021-25281) Detector](https://github.com/google/tsunami-security-scanner-plugins/tree/master/community/detectors/saltstack_cve_2021_25281)
```

Comment:
Please add a new line to the end of this file.

Reply:
done